pkg/detectors/openvpn/openvpn.go

```diff
@@ -0,0 +1,111 @@
+package openvpn
+
+import (
+	"context"
+	b64 "encoding/base64"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+
+	clientIdPat     = regexp.MustCompile(detectors.PrefixRegex([]string{"openvpn"}) + `\b([A-Za-z0-9-]{3,40}\.[A-Za-z0-9-]{3,40})\b`)
+	clientSecretPat = regexp.MustCompile(`\b([a-zA-Z0-9_-]{64,})\b`)
+	domainPat = regexp.MustCompile(`\b(https?:\/\/[A-Za-z0-9-]+\.api\.openvpn\.com)\b`) 
```

Comment:
suggestion: We can remove the redundant escape characters in this regex.
Ex: `\b(https?://[A-Za-z0-9-]+\.api\.openvpn\.com)\b` should work.